doc/declarations.md

```diff
@@ -528,9 +566,29 @@ by rw [div]; rfl
 end Hide
 ```
 
+If Lean cannot find a permutation of the arguments for which all recursive calls are decreasing, it will print a table that contains, for every recursive call, which arguments Lean could prove to be decreasing.
```

Comment:
Would it make sense to include an example table here? Or too difficult to keep in sync?

Reply:
Will do